subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/ivyservice/moduleconverter/dependencies/DefaultExcludeRuleConverterTest.java

```diff
@@ -47,4 +48,48 @@ public void testCreateExcludeRule() {
         assertThat(exclude.getConfigurations(),
                 Matchers.equalTo(Collections.singleton(configurationName)));
     }
+
+    @Test
+    @Issue("gradle/gradle#951")
+    public void testCreateExcludeRuleNullConfigurationName() {
```

Comment:
Would you mind turning this class into a Spock test? Using Spock's syntax we could roll both test cases into one which would make it much more manageable.